### PR TITLE
throw error when the language of COMM frame isn't ISO-639-2 encoded

### PR DESCRIPTION
**Pull request by m13m**

### CODE REVIEW

File: test/id3vx/encoding_test.exs [hunk 1]
```diff
@@ -288,10 +288,10 @@ defmodule Id3vx.EncodingTest do
...
 
     assert <<"COMM", frame_size::size(32), _flags::binary-size(2)>> = frame_header
-    assert 39 == frame_size
+    assert 35 == frame_size
 
     assert <<0x01::size(8), frame_rest::binary>> = frame_data
-    assert <<"english", frame_rest::binary>> = frame_rest
+    assert <<"eng", frame_rest::binary>> = frame_rest
 
     [content_description, content_text] = :binary.split(frame_rest, <<0, 0>>)
```
[the remark this answers is not in the document]
Comment (m13m) [reply test/id3vx/encoding_test.exs:297]: removed now we are pattern match to extract the frame_rest through encoding byte help.

**Comment by lawik** [comment]
Has conflicts. Otherwise ready?

**Comment by m13m** [comment]
@lawik yes.

**Comment by m13m** [comment]
@lawik this COMM frame is also review ready 

Both id3v2 and ffmpeg are able to parse the COMM frame of generated mp3.

**Pull request merged**